go_node_engine/addons/flops/FLOps.go

```diff
@@ -0,0 +1,52 @@
+package flops
+
+import (
+	"go_node_engine/logger"
+	"os/exec"
+	"strings"
+)
+
+func HandleFLOpsDataManager() {
+	ml_data_server_image := "ghcr.io/oakestra/addon-flops/ml-data-server:latest"
+	container_name := "ml_data_server"
+
+	cmd := exec.Command("docker", "ps", "-a", "--format", "{{.Names}}")
```

Comment:
Are we not using docker client in go, in order to have less dependencies thus reduced image size?

Reply:
I do not fully understand your question. The node-engine is a binary and not an image, so when you mention "image size" I assume you mean the size of the node-engine. Yes, it might get larger due to this change. However, trying to get this running via the containerd directly is a lot more complicated than just spinning up a docker side-car.

Let's see what @giobart take on this is. 🙂

(Note: I do not claim that this is the best solution, this is just something that I managed to get working as intended (in a reasonable amount of time.)